### ISSUE TITLE
eval.py generates plain black images

### ISSUE THREAD

**Issue by YoungJoongUNC** [YapengTian/TDAN-VSR#11]
Hello, thank you for sharing your great work! I have a question about eval.py.

I used vid4 test dataset calendar images for evaluation. I found that eval.py generates plain black images with x4 sizes. 

I tried to save images before applying converting to uint8  (eval.py last line), and it somehow generates calendar images but with wrong colors. 

May I ask how could I fix this?

Thank you.

**Comment by YapengTian** [comment]
Hi YoungJoong,

Thanks for your interest. I just did a test using video frames from the calendar set. It works well. I have uploaded the test images into the repo in the data/Vid4 folder (PLEASE change test data path accordingly when you do testing) and the results in the res folder. Could you please try the uploaded frames? I generated testing images using a script similar to https://github.com/yulunzhang/RDN/blob/master/RDN_IR/RDN_TestCode/RDN_SR_RGB/Prepare_TestData_HR_LR.m.  

If there are still some issues, please let me know.

Best,
Yapeng

**Comment by YoungJoongUNC** [comment]
Thank you for your fast reply!
I have tried change test data path to "data\Vid4" and now I got this error.

Traceback (most recent call last):
  File "eval.py", line 101, in <module>
    image = io.imread(os.path.join(LR, ims[0]))
"Could not find a format to read the specified file in %s mode" % modename
ValueError: Could not find a format to read the specified file in single-image mode

May I ask how could I fix this? I have found that when I set the test data path as "data", the network generates results well on SPMCs.
